### PR TITLE
:sparkles: Add chair Corsair T3 rush

### PR DESCRIPTION
**Pull request by RouxAntoine**
Advice about Corsair T3 rush after one year of use.

### CODE REVIEW

File: awesome-remote-equipment-fr.md [hunk 1]
```diff
@@ -49,6 +49,8 @@
 > « Un peu petit pour les personnes de plus d'1m80 »
 - [Fauteuil gamer corsair T1 2018](https://www.boulanger.com/ref/1114143) 
 > « Commandé et en cours de livraison, ce fauteuil a été recommandé sur Slack. Je le trouve plus abordable que du secret Lab par exemple »
+- [Fauteuil Corsair T3 Rush Tissu en Polyester](https://www.amazon.fr/gp/product/B07Y98Y4L6/)
+> « Je l'ai depuis un an bien en été car la matière ne tient pas trop chaud. Niveau confort c'est pas mal mais un peut trop droit. Point plus l'inclinaison 180° pour les siestes le midi 👌 »
```
Comment (jlandure) [inline awesome-remote-equipment-fr.md:53]:
```suggestion
> « Je l'ai depuis un an bien en été car la matière ne tient pas trop chaud. Niveau confort c'est pas mal mais un peu trop droit. Point en plus: l'inclinaison 180° pour les siestes le midi 👌 »
```

**Review by jlandure — CHANGES_REQUESTED** [review]

**Review by jlandure — APPROVED** [review]

**Comment by jlandure** [comment]
@all-contributors please add @RouxAntoine for doc

File: awesome-remote-equipment-fr.md (hunk 1 above)
Comment (RouxAntoine) [reply awesome-remote-equipment-fr.md]: Merci 👌

Comment (jlandure) [reply awesome-remote-equipment-fr.md]: De rien ! 🙌

**Pull request merged**